packages/frontend/src/features/sqlRunner/store/sqlRunnerSlice.ts

```diff
@@ -75,10 +62,42 @@ export const sqlRunnerSlice = createSlice({
                 }),
                 {},
             );
-
             state.resultsTableConfig = {
                 columns,
             };
+
+            // TODO: this initialization should be put somewhere it
+            // can be shared between the frontend and backend
+            state.chartConfig = {
+                metadata: {
+                    version: 1,
+                },
+                style: {
+                    legend: {
+                        position: 'top',
+                        align: 'center',
+                    },
+                },
+                axes: {
+                    x: {
+                        reference: Object.keys(action.payload[0])[0],
+                        label: Object.keys(action.payload[0])[0],
+                    },
+                    y: [
+                        {
+                            reference: Object.keys(action.payload[0])[1],
+                            label: Object.keys(action.payload[0])[1],
+                        },
+                    ],
+                },
```

Comment:
```suggestion
               const [xKey, yKey] = Object.keys(action.payload[0]);

const axes = {
    x: {
        reference: xKey,
        label: xKey,
    },
    y: [
        {
            reference: yKey,
            label: yKey,
        },
    ],
};
```

could do something like this, but just a nit-pick

Reply:
I did something similar